sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -83,13 +85,16 @@ private async ValueTask ProcessAsync(HttpMessage message, bool async)
 #if NET5_0
                 if (!async)
                 {
-                    responseMessage = _client.Send(httpRequest, HttpCompletionOption.ResponseHeadersRead, message.CancellationToken);
+                    // Sync HttpClient.Send is not supported on browser but so is the sync-over-async
```

Comment:
I've read this a few times and it doesn't quite make sense. Are you saying sync-over-async is *not* supported on the browser, or it is? Perhaps reword to clarify.